systemtest/src/test/java/io/strimzi/systemtest/security/PodSecurityProfilesST.java

```diff
@@ -197,7 +189,17 @@ void testOperandsWithRestrictedSecurityProfile() {
         verifyPodAndContainerSecurityContext(podsWithProperlyGeneratedSecurityCOntexts);
 
         LOGGER.info("Verify that Kafka cluster is usable and everything (MM1, MM2, and Connector) is working");
-        verifyStabilityOfKafkaCluster(testStorage);
+
+        // kafka cluster itself
+        final KafkaClients clusterClient = ClientUtils.getInstantPlainClientBuilder(testStorage)
+            .withPodSecurityPolicy(PodSecurityProfile.RESTRICTED)
+            .build();
```

Comment:
this and the previous clients are completely the same, so you don't need to create a new object here
plus I don't really understand the `//kafka cluster itself` comment